### DISCUSSION TITLE
Httpx

### DISCUSSION THREAD

**Discussion by Mathumitha97** [projectdiscovery/httpx#1957] [Q&A]
Httpx simply shows -dashboard is disabled I configured pdcp api key it keep on showing me invalid api try again.

**Comment by 3RassRK** [comment]
I was also getting the same issue
![httpxissue](https://github.com/user-attachments/assets/7bb5c543-b956-4fea-b34b-d5f9a00b7bd0)

**Comment by ehsandeep** [comment]
@Mathumitha97 

>  I configured pdcp api 

How you configured the API key?

**Comment by catbonny** [comment]
how you fix this im geeting same issue 
![image](https://github.com/user-attachments/assets/c216df67-a919-4ac8-a670-87167b9c7d24)

**Reply by GeorginaReeder** [reply]
Hi there @catbonny - as mentioned previously, have you configured your API key? If so, how was it configured exactly?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
